_posts/2019-01-25-updates.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Welcome to the New Way
+date: 2019-01-27  08:00:00
+categories: updates
+---
+__STATUS__:
+- current version: __3.60.0__
+- next scheduled release: __2/13/2019__
+
+Welcome to a new year, a new release, and a new set of development goals for the Ilios Project in the coming year! Key features identified as target milestones for our 2019-20 roadmap include:
+
+*SEARCH*: We have begun a year-long initiative to begin the incorporation of deep search. We expect to include search across the entire curriculum and its content.
+
+*Session Linking*: Session linking tools will be available in the next release.
```

Comment:
via @dartajax filtered though HemingwayApp.com:

```suggestion
*Session Linking*: The next release will allow linking ILMs to upcoming sessions. More session linking capabilities will be available in the future.
```

Reply:
👍

Reply:
I like @Trott 's change here. I apologize if I `resolved` the conversation. I think this change should be included not "resolved".